composer.json

```diff
@@ -16,19 +16,20 @@
         }
     ],
     "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
         "php": ">=5.3.9",
-        "symfony/framework-bundle": "~2.3.27 || ~2.6.6 || ~2.7 || ~3.0",
+        "symfony/framework-bundle": "^2.3.27 || ~3.0",
```

Comment:
I agree for 2.6.6, but I think it should be on conflict section.

Why 2.4 and 2.5 is not compatible?